build.sc

```diff
@@ -174,13 +174,20 @@ object dummy extends Module {
       Deps.ammonite
     )
   }
-  object scalafmt extends ScalaModule with Bloop.Module {
+  object bloop extends ScalaModule with Bloop.Module {
     def skipBloop    = true
     def scalaVersion = Scala.defaultInternal
     def ivyDeps = Agg(
       Deps.scalafmtCli
     )
   }
+  object scalafmt extends ScalaModule with Bloop.Module {
+    def skipBloop    = true
+    def scalaVersion = Scala.defaultInternal
+    def ivyDeps = Agg(
```

Comment:
Seems the `ivyDeps` of `bloop` and `scalafmt` were inverted.

Reply:
Thanks! I fixed it.